iothub/service/src/ServiceClient.cs

```diff
@@ -95,55 +150,184 @@ public static ServiceClient CreateFromConnectionString(string connectionString,
 
             var iotHubConnectionString = IotHubConnectionString.Parse(connectionString);
             bool useWebSocketOnly = transportType == TransportType.Amqp_WebSocket_Only;
-            var serviceClient = new AmqpServiceClient(iotHubConnectionString, useWebSocketOnly, transportSettings, options);
+            var iotHubConnection = new IotHubConnection(iotHubConnectionString, AccessRights.ServiceConnect, useWebSocketOnly, transportSettings);
+            var httpClientHelper = new HttpClientHelper(
+               iotHubConnectionString.HttpsEndpoint,
+               iotHubConnectionString,
+               ExceptionHandlingHelper.GetDefaultErrorMapping(),
+               s_defaultOperationTimeout,
+               transportSettings.HttpProxy);
+            var serviceClient = new ServiceClient(iotHubConnection, iotHubConnectionString.IotHubName, httpClientHelper, options);
             return serviceClient;
         }
 
         /// <summary>
-        /// Open the ServiceClient instance.
+        /// Open the ServiceClient instance. This call is executed over AMQP.
```

Comment:
We should avoid using the word 'executed' for the associated negative connotations. How about 'This call is made over AMQP.'?